src/managing-workflow/tuning-component-settings.md

```diff
@@ -19,6 +19,22 @@ env:
     value: "admin_only"
 ```
 
+## Setting Resource limits
+You can set resource limits to workflow components by modifying the template file `tpl/generate_params.toml`.
+This file has a section for each workflow component to set a limit to any workflow component just add limits_cpu, limits_memory
+in the section and set them to your appropriate values.
+
+Below is an example how builder section looks when CPU and Memory limits in `tpl/generate_params.toml`.
```

Comment:
Below is an example how the* `Builder`* (link to builder in components.md) section looks when CPU and memory* limits are set in `tpl/generate_params.toml`:*